docs/examples/kubo-as-a-library/go.mod

```diff
@@ -7,7 +7,7 @@ go 1.20
 replace github.com/ipfs/kubo => ./../../..
 
 require (
-	github.com/ipfs/boxo v0.13.0
+	github.com/ipfs/boxo v0.13.1-0.20230921121913-339ec17da598
```

Comment:
@hacdias are you going to do patch release?

Reply:
I'm gonna switch in follow up.